### ISSUE TITLE
Adding Student to Sections

### ISSUE THREAD

**Issue by scottholmes** [elmsln/elmsln#803]
Please excuse my denseness but I have yet to find a way to associate people records with course sections.  It's been quite some time since I coded in PHP, at least not anything more than some simple drupal modules.  What I fail to see is a reason you do not have a simple array of students as a part of the section content type.  There is a field for teacher and a field for TA.  I'm nearing the point where I will want to set up a public version of my course - limited to reading the course content and perhaps working the Summary Quizzes.  I would expect the visitors to log in as Guest or Student.  I would be satisfied by being able to add the students "by hand" as it would not be part of my demonstration.

**Comment by btopro** [comment]
there's a lot of issues calling for a People dashboard / adding this so that you don't NEED code but right now you need code in order to do this; sorry. #550 #798 #617 #253 all express the setup and getting things in position to add people to the people domain after mirroring the section data over to that domain. The reason for this is that CIS is short for course information system. It's for the logical routing / hooking things like SIS roster source data up to the section data in ELMSLN. People will be more for the "I just wnat to add student A to course B" kind of logic. There are capabilities to do this in #470 #494 Comply / compliance. Creating a System / Course request you'll be able to add someone to courses via the backend job runner. We'll get there for adding people too but it's not there currently.

**Comment by scottholmes** [comment]
As a follow up up, I've decided my pursuit of students is premature.  My real need at this time is to allow interested parties to view the course content and the general flow of what the course is about.  I have adjusted the permission for the "guest" role in the "course" service  by whittling down the permissions given to "staff".  This allows a login, leading to a list of courses, then the particular course I'm designing.  Of concern here is that "Bypass content access control" needs to be enabled as well as the three options for viewing content  and three options for course content.  The bypass control is noted to be a security risk.
